src/scss/components/_button.scss

```diff
@@ -133,6 +130,14 @@
     }
   }
 
+  &--full-card {
+    border-top: get-border(default);
+    border-bottom-right-radius: $borderRadius;
+    border-bottom-left-radius: $borderRadius;
```

Comment:
@MarcoBrunoBR I can't really change the name of $borderRadius on this PR because its the default variable for all the existing borders,  also, this change will affect files not related to this PR. We can discuss about the naming convention after the merge to v11-alpha.  Do you agree? @eduardojmatos @wbotelhos